assets/src/blocks/CarouselHeader/useSlides.js

```diff
@@ -78,20 +76,19 @@ export const useSlides = (slidesRef, lastSlide, containerRef, options = {
         container.style.height = null
       );
     }
-  };
+  }, [containerRef]);
 
-  const goToSlide = (newSlide, forceCurrentSlide = false) => {
+  const goToSlide = useCallback((newSlide, forceCurrentSlide = false) => {
     if (!slidesRef.current) {
       return;
     }
-
     const nextElement = slidesRef.current[newSlide];
     const activeElement = slidesRef.current[currentSlide];
 
-    if (newSlide !== currentSlide && nextElement && activeElement && !sliding) {
+    if (nextElement && activeElement && !sliding) {
```

Comment:
Why remove the `newSlide !== currentSlide` check?

Reply:
Because it will never be the same. See [goToSlide(currentSlide === 0 ? totalSlides - 1 : currentSlide - 1)](https://github.com/greenpeace/planet4-master-theme/pull/2618/files#diff-cc4c530ed86338a489136801860ce0d0e86bd5401be4e5f9e575ffe0e8641153R127) and [goToSlide((currentSlide + 1 >= totalSlides) ? 0 : currentSlide + 1)](https://github.com/greenpeace/planet4-master-theme/pull/2618/files#diff-cc4c530ed86338a489136801860ce0d0e86bd5401be4e5f9e575ffe0e8641153R131)